auth_token/utils.py

```diff
@@ -1,15 +1,17 @@
+import binascii
+import os
 import re
 
-from django.middleware.csrf import rotate_token
-from django.contrib.auth import load_backend, _get_backends
-from django.contrib.auth.signals import user_logged_in, user_logged_out
-from django.contrib.auth.models import AnonymousUser
 from django.conf import settings as django_settings
-
-from ipware.ip import get_ip
+from django.contrib.auth import _get_backends, load_backend
+from django.contrib.auth.models import AnonymousUser
+from django.contrib.auth.signals import user_logged_in, user_logged_out
+from django.middleware.csrf import rotate_token
+from django.utils.encoding import force_text
 
 from auth_token.config import settings
-from auth_token.models import Token, AnonymousToken
+from auth_token.models import AnonymousToken, Token
```

Comment:
tyhle bych přesunul dovnitř generate_key, ať se vyřeší ten potenciální kruhový import tady a nemusí se řešit všude, kde se tohle importuje 👍

Reply:
right, I did it that way. thx